### PR TITLE
Network vs node configuration 

### PR DESCRIPTION
**Pull request by MadelineMurray**
Network vs node configuration

### CODE REVIEW

File: docs/Configuring-Pantheon/Network-vs-Node.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+description: Configuring Pantheon at the network level compared to the node level 
+<!--- END of page meta data -->
+
+# Network vs Node Configuration 
+
+Pantheon is configured at the network level and the node level. 
+
+Network wide settings are specified in the genesis file.  For example, `difficulty` for Proof of Work
```
Comment (NicolasMassart) [inline docs/Configuring-Pantheon/Network-vs-Node.md]: The consensus algo would be a nice example too IMO.

File: docs/Configuring-Pantheon/Network-vs-Node.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+# Network vs Node Configuration 
+
+Pantheon is configured at the network level and the node level. 
+
+Network wide settings are specified in the genesis file.  For example, `difficulty` for Proof of Work 
+networks or `epochlength` for IBFT 2.0 networks. 
+
+Node settings are specified on the command line or in the [node configuration file](Using-Configuration-File.md). 
+For example, the JSON-RPC API methods to enable or the data directory for the node.
```
Comment (NicolasMassart) [inline docs/Configuring-Pantheon/Network-vs-Node.md]:
```suggestion
For example, the [JSON-RPC API methods](../Reference/JSON-RPC-API-Methods.md) to enable or the data directory for the node. 
```
I like hyper-links 😄

File: docs/Configuring-Pantheon/Network-vs-Node.md (hunk 1 above)
Comment (MadelineMurray) [reply docs/Configuring-Pantheon/Network-vs-Node.md]: Fixed

File: docs/Configuring-Pantheon/Network-vs-Node.md (hunk 2 above)
Comment (MadelineMurray) [reply docs/Configuring-Pantheon/Network-vs-Node.md]: Added

**Pull request merged**